IFC4x3/PropertyConstants/s/SPRAYED_WET_1K7hY64hDBQe59hbmdY$B6.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="SPRAYED_WET_1K7hY64hDBQe59hbmdY$B6" Name="SPRAYED_WET" UniqueId="541eb886-12b3-4b6a-8149-ae5c278bf2c6" />
```

Comment:
same source as above: "the concrete mix is watered and machine-mixed then pumped through a pipe to the application nozzle."

Reply:
Added description in [03a28cc](https://github.com/bSI-InfraRoom/IFC-Specification/pull/824/commits/03a28cc8c6229008d4d538438d999ed34110178f)